### DISCUSSION TITLE
Specifying the theme an image is shown to

### DISCUSSION THREAD

**Discussion by cagix** [McShelby/hugo-theme-relearn#432] [General]
Is there a way to display an image depending on the selected theme? For example, to display a grayscale variant of an image if a darker theme is used? 

In GitHub Markdown, an HTML snippet can be used to select the right image in the respective theme: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to

**Comment by McShelby** [comment]
Not right now. The way GitHub is using it, could also work if you allow HTML in your markdown but needs to written by the author.

Otherwise you could go with a shortcode.

Nevertheless, the GitHub way only checks for your OS settings and not for the selected theme.

**Reply by cagix** [reply]
A shortcode should do, thanks!

It's a pity that I cannot react to the selected theme of the generated page. But presumably people who have set the system to "dark" should be more likely to select a darker theme and vice versa - and then it would fit.

But then again, the ReLearn theme doesn't respond to OS settings, either, does it?

**Reply by McShelby** [reply]
The Relearn theme does not adhere to `prefers-color-scheme` and will not in the near future as it complicates configuration and manual selection of variants by the user.

**Reply by cagix** [reply]
Hmmm, that would make a great feature for a future release :)
